browser/mocha_tests/CanvasSectionContainer.test.ts

```diff
@@ -23,7 +23,7 @@
 var jsdom = require('jsdom');
 var assert = require('assert').strict;
 
-var dom = new jsdom.JSDOM(canvasDomString());
+var dom = new jsdom.JSDOM(canvasDomString(), { pretendToBeVisual: true });
```

Comment:
This doesn't seem to work, but as I understand it, should? I'd love to know what I'm missing here...

Reply:
Fixed.